src/installer/corehost/cli/test/nativehost/CMakeLists.txt

```diff
@@ -52,4 +52,6 @@ target_link_libraries(${DOTNET_PROJECT_NAME} nethost)
 # Specify non-default Windows libs to be used for Arm/Arm64 builds
 if (CLR_CMAKE_TARGET_WIN32 AND (CLR_CMAKE_TARGET_ARCH_ARM OR CLR_CMAKE_TARGET_ARCH_ARM64))
     target_link_libraries(${DOTNET_PROJECT_NAME} Advapi32.lib Ole32.lib OleAut32.lib)
+elseif(NOT CLR_CMAKE_TARGET_WIN32)
+    target_link_libraries(${DOTNET_PROJECT_NAME} ${CMAKE_DL_LIBS})
```

Comment:
Is the call to `set_common_libs` in `testexe.cmake` not sufficient for this?

Reply:
Good point. It was under `CLR_CMAKE_TARGET_LINUX` in `common.cmake`. I have updated the condition to include `OR CLR_CMAKE_TARGET_SUNOS`.